std/string.d

```diff
@@ -1914,21 +1935,23 @@ if (isSomeChar!Char && isSomeChar!Char2)
 }
 
 /**
-    Returns the index of the last occurrence of any of the elements in $(D
-    needles) in `haystack`. If no element of `needles` is found,
-    then `-1` is returned. The `stopIdx` slices `haystack` in the
-    following way $(D s[0 .. stopIdx]). `stopIdx` represents a codeunit
-    index in `haystack`. If the sequence ending at `startIdx` does not
-    represent a well formed codepoint, then a $(REF UTFException, std,utf) may be
-    thrown.
+    Searches the string `haystack` for the last occurrence of any of the
+    characters in `needles`.
 
     Params:
-        haystack = String to search for needles in.
-        needles = Strings to search for in haystack.
-        stopIdx = slices haystack like this $(D haystack[0 .. stopIdx]). If
-            the stopIdx is greater than or equal to the length of haystack the
-            functions returns `-1`.
-        cs = Indicates whether the comparisons are case sensitive.
+        haystack = string to search needles in
+        needles = characters to search for in `haystack`
+        stopIdx = index in `haystack` to stop searching at (exclusive); defaults
```

Comment:
There is no stopIdx parameter in the following function signature.
Am I missing something?

Reply:
sorry I was stupid